src/iqm/qiskit_iqm/iqm_provider.py

```diff
@@ -66,16 +66,22 @@ def _default_options(cls) -> Options:
 
     @property
     def max_circuits(self) -> Optional[int]:
+        """Set :attr:`max_circuits` property to a custom value.
+
+        Generally, this property is set automatically to `None` during the construction of backend instance. `None`
+        means there is no limit to the maximum number of circuits that can be run in a single batch.
```

Comment:
Can, or should? See the last sentence below.

Reply:
Could? :-)